### PR TITLE
Change the name from bitcoind to bgold

### PR DESCRIPTION
**Pull request by tan-kitty**

### CODE REVIEW

File: src/bitcoind-res.rc [hunk 1]
```diff
@@ -16,14 +16,14 @@ BEGIN
     BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
-            VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "bitcoind (Bitcoin node with a JSON-RPC server)"
+            VALUE "CompanyName",        "Bitcoin Gold"
+            VALUE "FileDescription",    "bgold (Bitcoin Gold node with a JSON-RPC server)"
             VALUE "FileVersion",        VER_FILEVERSION_STR
-            VALUE "InternalName",       "bitcoind"
+            VALUE "InternalName",       "bgold"
```
Comment (StarbuckBG) [inline src/bitcoind-res.rc]: bgoldd

File: src/bitcoind-res.rc [hunk 2]
```diff
@@ -16,14 +16,14 @@ BEGIN
...
             VALUE "LegalCopyright",     COPYRIGHT_STR
             VALUE "LegalTrademarks1",   "Distributed under the MIT software license, see the accompanying file COPYING or http://www.opensource.org/licenses/mit-license.php."
-            VALUE "OriginalFilename",   "bitcoind.exe"
-            VALUE "ProductName",        "bitcoind"
+            VALUE "OriginalFilename",   "bgold.exe"
```
Comment (StarbuckBG) [inline src/bitcoind-res.rc]: bgoldd.exe

File: src/bitcoind-res.rc [hunk 3]
```diff
@@ -16,14 +16,14 @@ BEGIN
...
+            VALUE "ProductName",        "bgold"
```
Comment (StarbuckBG) [inline src/bitcoind-res.rc]: bgoldd

Comment (tankitty) [reply src/bitcoind-res.rc]: thanks

**Comment by h4x3rotab** [comment]
lgtm

**Comment by tankitty** [comment]
@h4x3rotab  please check last commit

**Comment by h4x3rotab** [comment]
@tankitty Looks good but not enough to pass TravisCI tests:
https://travis-ci.org/BTCGPU/BTCGPU/jobs/290889862

**Comment by tankitty** [comment]
@h4x3rotab T_T

**Comment by Himan2001** [comment]
dont forget the renamings for the help and --help functions to rename it completly.
-> Question: it is a good idea to still use bitcoin.conf instead bgold.conf ?

**Comment by tankitty** [comment]
@h4x3rotab please help at last TravisCI test, thank you

**Comment by h4x3rotab** [comment]
Could you try to compile it locally and run unit test & regression tests?

**Comment by tankitty** [comment]
@h4x3rotab Yes, I compile on Debian 8 Jessie 

```
TEST                           | STATUS    | DURATION

abandonconflict.py             | P Passed  | 22 s
bip65-cltv-p2p.py              | P Passed  | 6 s
bip68-112-113-p2p.py           | P Passed  | 20 s
bipdersig-p2p.py               | P Passed  | 5 s
blockchain.py                  | P Passed  | 8 s
bumpfee.py                     | P Passed  | 15 s
decodescript.py                | P Passed  | 3 s
disablewallet.py               | P Passed  | 4 s
disconnect_ban.py              | P Passed  | 8 s
fundrawtransaction.py          | P Passed  | 37 s
getchaintips.py                | P Passed  | 11 s
httpbasics.py                  | P Passed  | 7 s
import-rescan.py               | P Passed  | 17 s
importmulti.py                 | P Passed  | 11 s
importprunedfunds.py           | P Passed  | 5 s
invalidblockrequest.py         | P Passed  | 4 s
invalidtxrequest.py            | P Passed  | 4 s
keypool-topup.py               | P Passed  | 12 s
keypool.py                     | P Passed  | 7 s
listsinceblock.py              | P Passed  | 12 s
listtransactions.py            | P Passed  | 40 s
mempool_limit.py               | P Passed  | 5 s
mempool_persist.py             | P Passed  | 21 s
mempool_reorg.py               | P Passed  | 10 s
mempool_resurrect_test.py      | P Passed  | 3 s
mempool_spendcoinbase.py       | P Passed  | 3 s
merkle_blocks.py               | P Passed  | 10 s
mining.py                      | P Passed  | 5 s
multi_rpc.py                   | P Passed  | 7 s
multiwallet.py                 | P Passed  | 6 s
net.py                         | P Passed  | 5 s
nulldummy.py                   | P Passed  | 3 s
p2p-compactblocks.py           | P Passed  | 32 s
p2p-fullblocktest.py           | P Passed  | 183 s
p2p-leaktests.py               | x Failed  | 18 s
p2p-mempool.py                 | P Passed  | 3 s
p2p-segwit.py                  | P Passed  | 70 s
p2p-versionbits-warning.py     | P Passed  | 8 s
preciousblock.py               | P Passed  | 8 s
prioritise_transaction.py      | P Passed  | 8 s
proxy_test.py                  | P Passed  | 10 s
rawtransactions.py             | P Passed  | 21 s
receivedby.py                  | P Passed  | 27 s
reindex.py                     | P Passed  | 12 s
resendwallettransactions.py    | P Passed  | 5 s
rest.py                        | P Passed  | 13 s
rpcnamedargs.py                | P Passed  | 3 s
segwit.py                      | P Passed  | 12 s
sendheaders.py                 | P Passed  | 27 s
signmessages.py                | P Passed  | 3 s
signrawtransactions.py         | P Passed  | 2 s
txn_clone.py                   | P Passed  | 11 s
txn_doublespend.py --mineblock | P Passed  | 11 s
uptime.py                      | P Passed  | 3 s
wallet-accounts.py             | P Passed  | 3 s
wallet-dump.py                 | P Passed  | 6 s
wallet-encryption.py           | P Passed  | 8 s
wallet-hd.py                   | P Passed  | 22 s
wallet.py                      | P Passed  | 78 s
walletbackup.py                | P Passed  | 127 s
zapwallettxes.py               | P Passed  | 12 s

```

failed on p2p-leaktests.py

**Comment by tankitty** [comment]
@h4x3rotab  OK all passed

`TEST                           | STATUS    | DURATION

abandonconflict.py             | P Passed  | 14 s
bip65-cltv-p2p.py              | P Passed  | 6 s
bip68-112-113-p2p.py           | P Passed  | 20 s
bipdersig-p2p.py               | P Passed  | 5 s
blockchain.py                  | P Passed  | 8 s
bumpfee.py                     | P Passed  | 18 s
decodescript.py                | P Passed  | 3 s
disablewallet.py               | P Passed  | 3 s
disconnect_ban.py              | P Passed  | 8 s
fundrawtransaction.py          | P Passed  | 41 s
getchaintips.py                | P Passed  | 11 s
httpbasics.py                  | P Passed  | 7 s
import-rescan.py               | P Passed  | 17 s
importmulti.py                 | P Passed  | 11 s
importprunedfunds.py           | P Passed  | 5 s
invalidblockrequest.py         | P Passed  | 4 s
invalidtxrequest.py            | P Passed  | 4 s
keypool-topup.py               | P Passed  | 11 s
keypool.py                     | P Passed  | 7 s
listsinceblock.py              | P Passed  | 12 s
listtransactions.py            | P Passed  | 50 s
mempool_limit.py               | P Passed  | 5 s
mempool_persist.py             | P Passed  | 21 s
mempool_reorg.py               | P Passed  | 6 s
mempool_resurrect_test.py      | P Passed  | 3 s
mempool_spendcoinbase.py       | P Passed  | 3 s
merkle_blocks.py               | P Passed  | 10 s
mining.py                      | P Passed  | 6 s
multi_rpc.py                   | P Passed  | 7 s
multiwallet.py                 | P Passed  | 6 s
net.py                         | P Passed  | 5 s
nulldummy.py                   | P Passed  | 3 s
p2p-compactblocks.py           | P Passed  | 31 s
p2p-fullblocktest.py           | P Passed  | 184 s
p2p-leaktests.py               | P Passed  | 8 s
p2p-mempool.py                 | P Passed  | 3 s
p2p-segwit.py                  | P Passed  | 73 s
p2p-versionbits-warning.py     | P Passed  | 8 s
preciousblock.py               | P Passed  | 8 s
prioritise_transaction.py      | P Passed  | 8 s
proxy_test.py                  | P Passed  | 10 s
rawtransactions.py             | P Passed  | 20 s
receivedby.py                  | P Passed  | 34 s
reindex.py                     | P Passed  | 12 s
resendwallettransactions.py    | P Passed  | 5 s
rest.py                        | P Passed  | 12 s
rpcnamedargs.py                | P Passed  | 3 s
segwit.py                      | P Passed  | 13 s
sendheaders.py                 | P Passed  | 27 s
signmessages.py                | P Passed  | 3 s
signrawtransactions.py         | P Passed  | 2 s
txn_clone.py                   | P Passed  | 12 s
txn_doublespend.py --mineblock | P Passed  | 12 s
uptime.py                      | P Passed  | 2 s
wallet-accounts.py             | P Passed  | 3 s
wallet-dump.py                 | P Passed  | 7 s
wallet-encryption.py           | P Passed  | 8 s
wallet-hd.py                   | P Passed  | 22 s
wallet.py                      | P Passed  | 76 s
walletbackup.py                | P Passed  | 131 s
zapwallettxes.py               | P Passed  | 12 s
zmq_test.py                    | o Skipped | 0 s

ALL                            | P Passed  | 1089 s (accumulated) 
Runtime: 279 s`

**Comment by tankitty** [comment]
@h4x3rotab  OK , I compiled on Debian 8 Jessie, All tests passed 

```
TEST                           | STATUS    | DURATION

abandonconflict.py             | P Passed  | 14 s
bip65-cltv-p2p.py              | P Passed  | 6 s
bip68-112-113-p2p.py           | P Passed  | 20 s
bipdersig-p2p.py               | P Passed  | 5 s
blockchain.py                  | P Passed  | 8 s
bumpfee.py                     | P Passed  | 18 s
decodescript.py                | P Passed  | 3 s
disablewallet.py               | P Passed  | 3 s
disconnect_ban.py              | P Passed  | 8 s
fundrawtransaction.py          | P Passed  | 41 s
getchaintips.py                | P Passed  | 11 s
httpbasics.py                  | P Passed  | 7 s
import-rescan.py               | P Passed  | 17 s
importmulti.py                 | P Passed  | 11 s
importprunedfunds.py           | P Passed  | 5 s
invalidblockrequest.py         | P Passed  | 4 s
invalidtxrequest.py            | P Passed  | 4 s
keypool-topup.py               | P Passed  | 11 s
keypool.py                     | P Passed  | 7 s
listsinceblock.py              | P Passed  | 12 s
listtransactions.py            | P Passed  | 50 s
mempool_limit.py               | P Passed  | 5 s
mempool_persist.py             | P Passed  | 21 s
mempool_reorg.py               | P Passed  | 6 s
mempool_resurrect_test.py      | P Passed  | 3 s
mempool_spendcoinbase.py       | P Passed  | 3 s
merkle_blocks.py               | P Passed  | 10 s
mining.py                      | P Passed  | 6 s
multi_rpc.py                   | P Passed  | 7 s
multiwallet.py                 | P Passed  | 6 s
net.py                         | P Passed  | 5 s
nulldummy.py                   | P Passed  | 3 s
p2p-compactblocks.py           | P Passed  | 31 s
p2p-fullblocktest.py           | P Passed  | 184 s
p2p-leaktests.py               | P Passed  | 8 s
p2p-mempool.py                 | P Passed  | 3 s
p2p-segwit.py                  | P Passed  | 73 s
p2p-versionbits-warning.py     | P Passed  | 8 s
preciousblock.py               | P Passed  | 8 s
prioritise_transaction.py      | P Passed  | 8 s
proxy_test.py                  | P Passed  | 10 s
rawtransactions.py             | P Passed  | 20 s
receivedby.py                  | P Passed  | 34 s
reindex.py                     | P Passed  | 12 s
resendwallettransactions.py    | P Passed  | 5 s
rest.py                        | P Passed  | 12 s
rpcnamedargs.py                | P Passed  | 3 s
segwit.py                      | P Passed  | 13 s
sendheaders.py                 | P Passed  | 27 s
signmessages.py                | P Passed  | 3 s
signrawtransactions.py         | P Passed  | 2 s
txn_clone.py                   | P Passed  | 12 s
txn_doublespend.py --mineblock | P Passed  | 12 s
uptime.py                      | P Passed  | 2 s
wallet-accounts.py             | P Passed  | 3 s
wallet-dump.py                 | P Passed  | 7 s
wallet-encryption.py           | P Passed  | 8 s
wallet-hd.py                   | P Passed  | 22 s
wallet.py                      | P Passed  | 76 s
walletbackup.py                | P Passed  | 131 s
zapwallettxes.py               | P Passed  | 12 s
zmq_test.py                    | o Skipped | 0 s

ALL                            | P Passed  | 1089 s (accumulated) 
Runtime: 279 s
```

**Comment by h4x3rotab** [comment]
I'm going to close and reopen this PR to trigger the TravisCI check.

**Comment by h4x3rotab** [comment]
Thank you very much. Seem this is the last file that needs to be fixed: `test/util/data/bitcoin-util-test.json`

**Comment by tankitty** [comment]
@h4x3rotab welcome : )

**Comment by tankitty** [comment]
@h4x3rotab  please check job log

**Comment by andre-amorim** [comment]
I T.H.I.N.K. we should keep the string 'bit', otherwise it could become some sort of Klingon language.

**Comment by h4x3rotab** [comment]
Sorry for the long review period. We've spent most of the time on core consensus changes. Would you like to rebase to the current HEAD? That could make this PR looks clearer so that it could be easier to solve TravisCI errors.

**Comment by tankitty** [comment]
@h4x3rotab  done

**Comment by wizzfile** [comment]
ok

**Comment by h4x3rotab** [comment]
Sorry for the long wait. I'm going to rebase the change and merge it today.

File: configure.ac [hunk 4]
```diff
@@ -8,7 +8,7 @@ define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2017)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Gold]])
-AC_INIT([Bitcoin Gold],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin],[https://bitcoincore.org/])
```
Comment (ducminhgl3) [inline configure.ac]: 14tZxzG74xWxp4Eg7CGKag5TBUhVh5Xjyx

File: src/bitcoind-res.rc [hunk 5]
```diff
@@ -16,14 +16,14 @@ BEGIN
     BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
-            VALUE "CompanyName",        "Bitcoin"
```
Comment (ducminhgl3) [inline src/bitcoind-res.rc]: 14tZxzG74xWxp4Eg7CGKag5TBUhVh5Xjyx

File: src/bitcoind-res.rc [hunk 6]
```diff
@@ -16,14 +16,14 @@ BEGIN
...
-            VALUE "FileDescription",    "bitcoind (Bitcoin node with a JSON-RPC server)"
+            VALUE "CompanyName",        "Bitcoin Gold"
```
Comment (ducminhgl3) [inline src/bitcoind-res.rc]: 14tZxzG74xWxp4Eg7CGKag5TBUhVh5Xjyx

File: src/bitcoind-res.rc [hunk 7]
```diff
@@ -16,14 +16,14 @@ BEGIN
...
+            VALUE "FileDescription",    "bgold (Bitcoin Gold node with a JSON-RPC server)"
             VALUE "FileVersion",        VER_FILEVERSION_STR
```
Comment (ducminhgl3) [inline src/bitcoind-res.rc]: 14tZxzG74xWxp4Eg7CGKag5TBUhVh5Xjyx

File: src/bitcoind-res.rc (hunk 1 above)
Comment (wizzfile) [reply src/bitcoind-res.rc]: ок

**Pull request closed without merging**